tests/pkg_cn-cbor/main.c

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
with the unittests in place, I think the test is not needed anymore...

Reply:
Alright, noted!

Reply:
still here?

Reply:
yep

Reply:
Yep, seems so. let me check...